src/styles/main.scss

```diff
@@ -0,0 +1,75 @@
+$width-day: 100px;
+$gap: 1px;
+$start-day: (
+  mon 0,
+  tue ($width-day) + $gap,
+  wed ($width-day * 2) + ($gap * 2),
+  thu ($width-day * 3) + ($gap * 3),
+  fri ($width-day * 4) + ($gap * 4),
+  sat ($width-day * 5) + ($gap * 5),
+  sun ($width-day * 6) + ($gap * 6)
+);
+
+body {
+  height: 100%;
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($width-day * 7) + ($gap * 6);
+  padding: 10px 0;
+  gap: $gap;
```

Comment:
The code style guidelines require a space before the opening brace `{`. Please add a space on line 25 before `{` to adhere to the style rules.